### PR TITLE
Bump back to alpha post 1.20.0 release

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
     <AssemblyVersion>1.20.0</AssemblyVersion>
-    <InformationalVersion>1.20.0</InformationalVersion>
+    <InformationalVersion>1.20.0-alpha</InformationalVersion>
```
Comment (z1c0) [inline src/Directory.Build.props:6]:
```suggestion
    <InformationalVersion>1.21.0-alpha</InformationalVersion>
```

File: src/elastic_apm_profiler/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "elastic_apm_profiler"
-version = "1.20.0"
+version = "1.20.0-alpha"
```
Comment (z1c0) [inline src/elastic_apm_profiler/Cargo.toml:3]:
```suggestion
version = "1.21.0-alpha"
```

**Review by z1c0 — CHANGES_REQUESTED** [review]
This should be the incremented version + alpha suffix.

**Comment by Mpdreamz** [comment]
Doh thanks for catching that @z1c0 updated the other assembly version attributes too.

**Review by z1c0 — APPROVED** [review]

**Pull request merged**